src/components/CustomerQuota/CustomerCard.tsx

```diff
@@ -98,15 +99,20 @@ export const CustomerCard: FunctionComponent<{
       <Feather name="user" size={size(3)} color={color("grey", 0)} />
       <View style={styles.headerText}>
         <AppText style={styles.idLabel}>
-          Identification number{ids.length > 1 ? "s" : ""}
+          {i18n.t("customerQuotaScreen.quotaRedeemedLimitReachedIDNumber")}
```

Comment:
What about the extra s suffix logic?

```
{ids.length > 1 ? "s" : ""}
```

Reply:
Ohh I realised that there were some translations like `checkoutSuccessScreen.redeemedItems = "Item(s) redeemed"` and `merchantFlowScreen.quotaCategoryVouchers =  "Voucher(s)"` so for this I did `customerQuotaScreen.quotaRedeemedLimitReachedIDNumber = "Identification number(s)"` since having the logic will be a bit weird with the chinese translation part too.

Reply:
Oh ok can you let Eida know about this?